lib/generators/spotlight/templates/catalog_controller.rb

```diff
@@ -16,6 +16,7 @@ class CatalogController < ApplicationController
     # Blacklight 8 sets a default value to 'advanced'
     config.json_solr_path = nil
     config.header_component = Spotlight::HeaderComponent
+    config.exhibit_navbar_component = Spotlight::ExhibitNavbarComponent
```

Comment:
Think this is config.index?

Reply:
yes! thanks.

Reply:
I guess the question is, does it make sense to configure a different navbar depending on whether you're in the index or show view, or should this be configurable at the app level like the header component?

Reply:
I don't think it matters, but either this or this https://github.com/projectblacklight/spotlight/pull/3171/files#diff-de18dc9392ca38402d097f5d638e554a17c8b0ba6b2e341fe46ca6188e05cf1cR9 line need to be updated to be inline with each other.